Objects/codeobject.c

```diff
@@ -1370,6 +1370,15 @@ _PyCode_GetCode(PyCodeObject *co)
     return code;
 }
 
+PyObject *
+PyCode_GetCode(PyObject *co)
+{
+    if (!PyCode_Check(co)) {
+        PyErr_BadInternalCall();
```

Comment:
I forgot if this or a `TypeError` is preferred for the C API (or no error setting at all?)

Reply:
Use `PyCode_GetCode(PyCodeObject *co)` and then you won't need the check.

Reply:
I would like to, but `co_code` was previously `PyObject*`, and I want this to be a drop-in replacement as far as possible.

Reply:
PyErr_BadInternalCall() is fine, but I would even suggest replacing a runtime check with an assertion. The caller is responsible to pass the right type.